develop-docs/application-architecture/dynamic-sampling/the-big-picture.mdx

```diff
@@ -6,29 +6,37 @@ sidebar_order: 1
 
 ![Sequencing](./images/sequencing.png)
 
+
+<Alert title="💡 Note" level="info">
+
+Dynamic Sampling currently operates on either spans or transactions, based on the feature flag `dynamic-sampling-spans`. The logic between the two event types is similar, so most of this documentation is kept at a generic level and important differences are pointed out explicitly. 
+
+</Alert>
+
+
 ## Sequencing
 
 Dynamic Sampling occurs at the edge of our ingestion pipeline, precisely in [Relay](https://github.com/getsentry/relay).
 
-When transaction events arrive, in a simplified model, they go through the following steps (some of which won't apply if you self-host Sentry):
+When events arrive, in a simplified model, they go through the following steps (some of which won't apply if you self-host Sentry):
```

Comment:
Reading through it again, all of these apply in a self-hosted environment by now. We can remove this remark. 

```suggestion
When events arrive, in a simplified model, they go through the following steps:
```